screens/modals/ResponderHelpRequestModal.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component, useEffect } from "react";
+import {
+  StyleSheet,
+  Button
```

Comment:
I think Maxine made a default button we use which is in ```'components/buttons'```

Reply:
yea i know, its just a stub screen